lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
```

Comment:
why not to have something in the filename to refer to qe-sap-deployment?

Reply:
Depends if the intention is to have sles4sap specific basetest or it should be a general one for all qesap deployments.
qesap_publiccloud_basetest.pm ?